examples/DatePicker.basic.jsx

```diff
@@ -3,16 +3,36 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { DatePicker } from '@itwin/itwinui-react';
+import { DatePicker, IconButton, Popover } from '@itwin/itwinui-react';
+import { SvgCalendar } from '@itwin/itwinui-icons-react';
 
 export default () => {
   const [currentDate, setCurrentDate] = React.useState(new Date());
+  const [visible, setVisible] = React.useState(false);
   return (
-    <DatePicker
-      date={currentDate}
-      onChange={(date) => {
-        setCurrentDate(date);
-      }}
-    />
+    <Popover
+      content={
+        <DatePicker
+          date={currentDate}
+          onChange={(date) => {
+            setCurrentDate(date);
+            setVisible(false);
+          }}
+          setFocus
+        />
+      }
+      placement='bottom'
```

Comment:
Can we change all these to "bottom-start"? Feels a bit weird when it's center aligned

Reply:
Will do.